src/sagas/networkSettings.js

```diff
@@ -217,31 +238,42 @@ function invalidUrl(tryUrl) {
 export function* persistNetworkSettings(action) {
   // persists after reducer being updated
   const networkSettings = action.payload;
-  const strNetworkSettings = JSON.stringify(networkSettings);
-  yield call(AsyncStorage.setItem, networkSettingsKeyMap.networkSettings, strNetworkSettings);
-
-  // trigger toggle update to be managed by featureToggle saga
-  yield put(featureToggleUpdate());
```

Comment:
You don't need this anymore?

Reply:
No, because the feature toggle will be reinitialized in the process of `reloadWalletRequested()`.